tests/integration/targets/azure_rm_postgresqlflexibleserver/tasks/main.yml

```
@@ -150,8 +196,23 @@
       - output.servers[0].maintenance_window.day_of_week == 6
       - output.servers[0].maintenance_window.start_hour == 10
       - output.servers[0].maintenance_window.start_minute == 6
+      - user_identity_1 in output.servers[0].identity.user_assigned_identities
+      - user_identity_2 in output.servers[0].identity.user_assigned_identities
 
-- name: Create a post gresql flexible database(check mode)
+- name: postgresql server Idenity SystemAssigned
```

All names should start with an uppercase letter
```suggestion
- name: Postgresql server Idenity SystemAssigned
```

---

done